specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/examples/event-grid-schema/Resources_CreatedOrUpdated.json

```diff
@@ -0,0 +1,73 @@
+{
+  "id": "4eef929a-a65c-47dd-93e2-46b8c17c6c17",
+  "topic": "/subscriptions/{subscription-id}",
+  "subject": "/subscriptions/{subscription-id} /resourceGroups/{rg-name}/providers/Microsoft.Storage/storageAccounts/{storageAccount-name}",
```

Comment:
nit: no spaces in subject, data->resourceinfo->id

Reply:
done!